sabnzbd/nzbstuff.py

```diff
@@ -743,7 +743,11 @@ def __init__(
         else:
             # Determine "incomplete" folder
             self.download_path = os.path.join(cfg.download_dir.get_path(), self.work_name)
-            self.download_path = long_path(get_unique_dir(self.download_path, create_dir=True))
+            self.download_path = get_unique_dir(self.download_path, create_dir=True)
+            if not self.download_path:
+                logging.warning(T("Could not create download path: %s"), self.download_path)
```

Comment:
Doesn't create_all_dirs already throw AN ERROR? 
https://github.com/sabnzbd/sabnzbd/pull/2516/files#diff-db363ac9b90f46184c420b28c372136cff39bb5de103238aae125e0d1f4a1471R732

Reply:
Yeah. I was following the convention established elsewhere where an error log is written if the value is false.

https://github.com/sabnzbd/sabnzbd/blob/d22f2296c24ea2c705ad5109a9e53214bdb32068/sabnzbd/postproc.py#L701-L709

Reply:
I don't want to introduce a new translation just for this problem. Let's keep it simple:
```suggestion
            self.download_path = long_path(get_unique_dir(self.download_path, create_dir=True))
            if not self.download_path:
                raise NzbEmpty
```
Also I noticed it should be `raise` and not `return`.